Library/Formula/c.rb

```diff
@@ -0,0 +1,16 @@
+class C < Formula
+  homepage "https://github.com/ryanmjacobs/c"
+  url "https://github.com/ryanmjacobs/c/archive/v0.06.tar.gz"
+  sha1 "0eefb3523abccea7ebdd2c56d4d7be6f23585645"
+
+  depends_on "gnu-sed"
+
+  def install
+    inreplace "c", "sed", "gsed"
```

Comment:
We shouldn't need the `inreplace` anymore, fixed w/ https://github.com/ryanmjacobs/c/commit/6fd26113edbb75db80356122e8a6dbbd7e4529c3.

I'll bump the version to v0.7 so this will include those changes.